src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -141,15 +103,34 @@ export class SbUserComponent implements OnInit {
     });
   }
 
-    editUser(user: OrganizationsUsersList) {
-        this.dialogService.open(AddEditUserComponent, {
-            data: user,
-            header: this.i18nextPipe.transform('USER_EDIT'),
-            width: '30%',
-            height: 'auto'
-        });
+  loadUserList(event: any) {
+    let filters = this.filteredValue;
+    Object.keys(filters).forEach(key => {
+      if (!filters[key]) {
+        delete filters[key]
+      }
+    });
+
+    const body = {
+      request: {
+        filters: filters,
+        limit: event?.rows,
+        offset: event?.first ? (event?.first / 10) + 1 : 0,
+      }
     }
 
+    this.userService.loadUserList(body).subscribe(users => {
+      this.organizationsUsersList = users?.result?.response?.content;
+      this.count = users?.result?.response?.count;
+      this.loading = false;
+    }, (error: any) => {
+      this.loading = false;
+      this.messages = [];
+      this.messageService.add({ severity: 'error', detail: error.error.params.errmsg });
```

Comment:
Use the optional chaining (?.) operator for language constants also

Reply:
added